### PR TITLE
Add usage of Volcano helm repo to Readme

### PR DESCRIPTION
**Pull request by wangyang0616**
Add the basic usage instructions of Volcano helm repo in Readme.

associate: [#2766](https://github.com/volcano-sh/volcano/issues/2766)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+Once Helm is set up properly, add the repo as follows:
+
+```console
+helm repo add volcano-sh https://volcano-sh.github.io/helm-charts
+```
+
+You can then run `helm search repo volcano-sh` to see the charts.
+
+
+**Install Volcano:**
```
Comment (william-wang) [inline README.md:19]: Is there any possible to specify the version?

Comment (wangyang0616) [reply README.md:19]: When `helm install`, you can use `--version` to specify the version name, for example: `--version 1.6.0`, if not specified, the latest version will be installed by default.

Comment (wangyang0616) [reply README.md:19]: As far as I understand, there is no need to specify the version number here, just use the latest version to install it.

**Review by william-wang — APPROVED** [review]
/lgtm
/approve

**Comment by william-wang** [comment]
Fix the proboot issue and review again

**Review by william-wang — APPROVED** [review]
/lgtm

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**